packages/components/pie-avatar/src/defs.ts

```diff
@@ -1,3 +1,25 @@
-// TODO - please remove the eslint disable comment below when you add props to this interface
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface AvatarProps {}
+import { type ComponentDefaultProps } from '@justeattakeaway/pie-webc-core';
+
+export const tags = ['a', 'button', 'div'] as const;
+
+export interface AvatarProps {
+    label?: string;
+    /**
+     * Label for the username that will be turned into initials inside the avatar, if provided.
+     */
+    tag: typeof tags[number];
+    /**
+     * What HTML element the avatar should be such as button, a  or div.
+     */
```

Comment:
Comments should be added before its types:

```suggestion
    /**
     * Label for the username that will be turned into initials inside the avatar, if provided.
     */
    label?: string;
    /**
     * What HTML element the avatar should be such as button, a  or div.
     */
    tag: typeof tags[number];
```